synthtool/gcp/templates/java_library/.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -3,7 +3,13 @@ name: Bug report
 about: Create a report to help us improve
 
 ---
-
+{% if metadata['repo']['library_type'] == 'GAPIC_AUTO'
```

Comment:
unblocking suggestion:

It seems like the same set of `if` clauses exist in many places. Maybe you can make it a helper function receiving the metadata['repo'] and returns bool so that all the templates can use that helper function?

Reply:
Moved to `migrated_split_repo` variable in java.py. Thanks.